### ISSUE TITLE
Damon review

### ISSUE THREAD

**Issue by DamengRandom** [leahhou/leah-feelix-project#3]
Note 1: Improve readme file, eg: add notes for letting people know how to start the app locally, how to test the app and etc.

Note 2: `:global body`, I thought we can use `:global(.classname)`. In here, how about we just use `body`?

Note 3: New contact form field might need to add form validation, eg: I can enter `damon` as phone number, which is invalid

Note 4: For writing the React component, you many give a try to write some functional components next time (You will love it)

Note 5: Instead of `<Input></Input>` I suggest to use `<Input />`

Note 6: I suggest `id="searchInput"` instead of `id="search"`, we need a bit more unique id

Note 7: when I clicked on `Cancel` button from UI, I got this error message: `TypeError: this.props.handleClick is not a function`, please fix it. If you need help, I am willing to share the answer. 

Note 8: In `ContactForm.jsx` file, I suggest from 39 to line 72, we write a loop for each input field instead of write `<input/>` repeatedly (Reference: [DRY rule](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself))

Note 9: I saw some files you use `prop-types`, some files you didn't, its better to use for all components, keep it consistant

Note 10: I suggest to remove all `console.log` inside your codebase

In conclusion, very good start 👍  and keep going Leah ~~

**Comment by leahhou** [comment]
- [x] Note 1: Improve readme file, eg: add notes for letting people know how to start the app locally, how to test the app and etc.

- [ ] Note 2: :global body, I thought we can use :global(.classname). In here, how about we just use body?
**Can you explain a bit more why?** 

- [x] Note 3: New contact form field might need to add form validation, eg: I can enter damon as phone number, which is invalid

- [ ] Note 4: For writing the React component, you many give a try to write some functional components next time (You will love it)
**I start with class component because I find I am less familiar with them. definitely will try later!** 

- [x] Note 5: Instead of `<Input></Input>` I suggest to use `<Input />`
**Can you explain a bit more why?** 

- [x] Note 6: I suggest id="searchInput" instead of id="search", we need a bit more unique id
**used shortid npm to generate random id** 

- [x] Note 7: when I clicked on Cancel button from UI, I got this error message: TypeError: this.props.handleClick is not a function, please fix it. If you need help, I am willing to share the answer.
**fixed by just adding button toggle, later will try react routing** 

- [ ] Note 8: In ContactForm.jsx file, I suggest from 39 to line 72, we write a loop for each input field instead of write <input/> repeatedly (Reference: DRY rule)
**Though after add validation, I feel it is hard to do loop given each input field is quit specific?** 

- [x] Note 9: I saw some files you use prop-types, some files you didn't, its better to use for all components, keep it consistant

- [x] Note 10: I suggest to remove all console.log inside your codebase
